Assets/Mirror/Runtime/Transport/MultiplexTransport.cs

```diff
@@ -162,6 +162,14 @@ void InitServer()
             }
         }
 
+        // for now returns the first uri,
+        // should we return all available uris?
+        public override Uri ServerUri()
+        {
+            return transports[0].ServerUri();
```

Comment:
What if Websockets is the first transport and Telepathy is second?

Reply:
well then,  it will give you websocket url.

But suppose you have multiplex with WS and telepathy both.  If you want to make sure the discovery uses telepathy,  then drag telepathy in the NetworkDiscovery and you are done.